main.go

```diff
@@ -62,6 +63,11 @@ func main() {
 		logger.Fatal(err)
 	}
 
+	defaultPATeam, err := strconv.Atoi(defaultPATeamString)
+	if err != nil{
+		logger.Print("Error converting defaultPaString to int")
```

Comment:
I'd suggest changing `defaultPaString` to `DEFAULT_PA_TEAM` so it is clear what the issue was